test_responses.py

```diff
@@ -990,3 +990,76 @@ def test_cookies_from_headers():
     for k, v in cookiejar.items():
         assert isinstance(v, str)
         assert v == expected[k]
+
+
+def test_response_match_has_own_calls():
+    @responses.activate
+    def run():
+        match = responses.add(method=responses.GET, url="http://example.com", body="OK")
+        requests.get("http://example.com")
+        resp = match.calls[0].response
```

Comment:
It would be better if this test created two responses and called both of them. That would help ensure that each stub tracked responses separately. Right now that _could_ be true, but with no other stub we can't be sure.

Reply:
Test was extended 6b085bd